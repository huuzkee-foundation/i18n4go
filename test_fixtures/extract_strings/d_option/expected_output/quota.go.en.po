# filename: d_option/input_files/quota/quota.go, offset: 773, line: 30, column: 16
msgid "quota"
msgstr "quota"

# filename: d_option/input_files/quota/quota.go, offset: 797, line: 31, column: 16
msgid "CF_NAME quota QUOTA"
msgstr "CF_NAME quota QUOTA"

# filename: d_option/input_files/quota/quota.go, offset: 1263, line: 48, column: 13
msgid "Getting quota %s info as %s..."
msgstr "Getting quota %s info as %s..."

# filename: d_option/input_files/quota/quota.go, offset: 1521, line: 58, column: 21
msgid "Memory"
msgstr "Memory"

# filename: d_option/input_files/quota/quota.go, offset: 1756, line: 61, column: 21
msgid "Paid service plans"
msgstr "Paid service plans"

# filename: d_option/input_files/quota/quota.go, offset: 835, line: 32, column: 16
msgid "Show quota info"
msgstr "Show quota info"

# filename: d_option/input_files/quota/quota.go, offset: 1063, line: 38, column: 33
msgid "quotas"
msgstr "quotas"

# filename: d_option/input_files/quota/quota.go, offset: 1614, line: 59, column: 21
msgid "Routes"
msgstr "Routes"

# filename: d_option/input_files/quota/quota.go, offset: 1707, line: 60, column: 45
msgid "%d"
msgstr "%d"

# filename: d_option/input_files/quota/quota.go, offset: 1683, line: 60, column: 21
msgid "Services"
msgstr "Services"

