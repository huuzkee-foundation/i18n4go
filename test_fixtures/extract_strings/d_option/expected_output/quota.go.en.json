[{"id":"Show quota info","translation":"Show quota info"},{"id":"quotas","translation":"quotas"},{"id":"Routes","translation":"Routes"},{"id":"%d","translation":"%d"},{"id":"Services","translation":"Services"},{"id":"quota","translation":"quota"},{"id":"CF_NAME quota QUOTA","translation":"CF_NAME quota QUOTA"},{"id":"Getting quota %s info as %s...","translation":"Getting quota %s info as %s..."},{"id":"Memory","translation":"Memory"},{"id":"Paid service plans","translation":"Paid service plans"}]
